galsim/chromatic.py

```diff
@@ -679,10 +689,14 @@ def transform(self, dudx, dudy, dvdx, dvdy):
             _dudy = dudy
             _dvdx = dvdx
             _dvdy = dvdy
-            if not hasattr(dudx, '__call__'): _dudx = lambda w: dudx
-            if not hasattr(dudy, '__call__'): _dudy = lambda w: dudy
-            if not hasattr(dvdx, '__call__'): _dvdx = lambda w: dvdx
-            if not hasattr(dvdy, '__call__'): _dvdy = lambda w: dvdy
+            if not hasattr(dudx, '__call__'):
+                _dudx = lambda w: dudx
+            if not hasattr(dudy, '__call__'):
+                _dudy = lambda w: dudy
+            if not hasattr(dvdx, '__call__'):
+                _dvdx = lambda w: dvdx
+            if not hasattr(dvdy, '__call__'):
+                _dvdy = lambda w: dvdy
```

Comment:
Here's an example where the one-liners are more readable.  Should revert this change.